spec/features/dashboard/show_spec.rb

```diff
@@ -17,11 +17,11 @@
           gardens: {
             data: [ {id: '3', type: 'garden'}, {id: '4', type: 'garden'}] }}})
 
-      garden1 = File.read('spec/fixtures/public_garden.json')
-      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/gardens/3").to_return(status: 200, body: garden1)
+      @garden1 = File.read('spec/fixtures/public_garden.json')
+      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/gardens/3").to_return(status: 200, body: @garden1)
 
-      garden2 = File.read('spec/fixtures/private_garden.json')
-      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/gardens/4").to_return(status: 200, body: garden2)
+      @garden2 = File.read('spec/fixtures/private_garden.json')
```

Comment:
Looks like we aren't using these variables anywhere within tests so they don't need to be instance variables but no big deal.

Reply:
Oh yeah. I was trying to reference these in the tests so as to not hard code stuff but it was throwing errors for some reason.